apps/desktop/src/lib/vbranches/branchController.ts

```diff
@@ -272,9 +272,11 @@ export class BranchController {
 			await this.vbranchService.refresh();
 			return upstreamRef;
 		} catch (err: any) {
-			posthog.capture('Push Failed', { error: err });
 			console.error(err);
-			if (err.code === 'errors.git.authentication') {
+			const { code, message } = err;
+			posthog.capture('Push Failed', { code, message });
```

Comment:
Actually, I made a mistake here. It should be `posthog.capture('Push Failed', { error: { code, message }})`.